Gateway/Http/Client/TransactionPayOrInInstallments.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the MIT License
+ * It is available through the world-wide-web at this URL:
+ * https://tldrlegal.com/license/mit-license
+ * If you are unable to obtain it through the world-wide-web, please email
+ * to support@buckaroo.nl, so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade this module to newer
+ * versions in the future. If you wish to customize this module for your
+ * needs please contact support@buckaroo.nl for more information.
+ *
+ * @copyright Copyright (c) Buckaroo B.V.
+ * @license   https://tldrlegal.com/license/mit-license
+ */
+declare(strict_types=1);
+
+namespace Buckaroo\Magento2\Gateway\Http\Client;
+
+use Buckaroo\Transaction\Response\TransactionResponse;
+
+class TransactionPayOrInInstallments extends DefaultTransaction
+{
+    /**
+     * @inheritdoc
+     */
+    protected function process(string $paymentMethod, array $data): TransactionResponse
+    {
+        $this->action = TransactionType::PAY_IN_INSTALLMENTS;
```

Comment:
why do you store this in a class variable?

Reply:
I modified, I wanted to keep the same structure as on DefaultTransaction and maybe we will need it outside the process function.